### ISSUE TITLE
Error during the upgrade from 24.0.4.1 to 24.0.5.1

### ISSUE THREAD

**Issue by gloolim** [nextcloud/all-in-one#1164]
### Steps to reproduce
1. Automatic backup start

### Expected behavior 
Nextcloud docker doesn't start due to failed upgrade app "updatenotification".

### Actual behavior 
Successful update
I restore several times from backup but every day when automatic backup start the instance crash. Now I've disabled the automatic backup.

### Host OS 
QNAP

#### Nextcloud AIO version 
v2.0.2

#### Current channel 
latest

#### Other valuable info 

Logs from nextcloud docker
```
nc: getaddrinfo: Try again
Waiting for database to start...
             now              
------------------------------
 2022-09-18 09:38:50.98243+02
(1 row)

Configuring Redis as session handler...
Setting php max children...
Checking connection to appstore
Curl didn't produce a 200 status, is appstore reachable?
Checking connection to appstore
Curl didn't produce a 200 status, is appstore reachable?
Checking connection to appstore
Appstore is reachable
Maintenance mode already disabled
Getting and backing up the status of apps for later, this might take a while...
Initializing nextcloud 24.0.5.1 ...
Initializing finished
Upgrading nextcloud from 24.0.4.1 to 24.0.5.1...
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Setting log level to debug
Turned on maintenance mode
Updating database schema
Updated database
UnexpectedValueException: The files of the app "updatenotification" were not correctly replaced before running the update
Update failed
Maintenance mode is kept active
Resetting log level
Upgrade failed. Please restore from backup.
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
```

**Comment by szaimen** [comment]
Hi, was this Nextcloud instance migrated to AIO by following https://github.com/nextcloud/all-in-one/blob/main/migration.md#migrate-the-files-and-the-database?

**Comment by gloolim** [comment]
Yes, I migrated from VM instance to AIO instance by following this how-to.

**Comment by szaimen** [comment]
Okay, then I have an idea what could trigger this behaviour and how to resolve it in a future update

**Comment by szaimen** [comment]
This is now fixed with v2.0.3 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
